app/helpers/application_helper.rb

```diff
@@ -441,10 +441,11 @@ def accessible_resource_records(resource, order = :name)
     klass.authorized.reorder(order)
   end
 
-  def accessible_resource(obj, resource, order = :name)
+  def accessible_resource(obj, resource, order = :name, opts = {})
```

Comment:
you can use a keyward arg that defaults to resource here instead of a option hash